specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-03-01/workspace.json

```diff
@@ -1393,6 +1401,16 @@
         "value"
       ],
       "description": "The response to a list restorable dropped Sql pools request"
+    },
+    "NetworkSettings": {
+      "description": "Network Settings",
+      "type": "object",
+      "properties": {
+        "publicNetworkAccess": {
+          "description": "Enable or Disable pubic network access to workspace",
```

Comment:
From the description, looks like this property is kind of a boolean -> Enable/Disable. Any reason why it is modeled as a string? If modeling as string required, can this be modeled as enum?
Please refer http://azure.github.io/autorest/extensions/#x-ms-enum

Reply:
consider using name something like "publicNetworkAccessState".

Reply:
changed it to enum